tests/test_ruby.sh

```diff
@@ -43,4 +52,25 @@ function test_rbenv {
   done
 }
 
+function test_rvm {
+
+  rvm-prompt() {
+    printf '%s\n' "ruby-3.2.2"
+  }
+
+  rvm() {
+    printf '%s\n' "$__rvm_output"
+  }
+
+  _LP_RUBY_VENV_PROGRAM=rvm
+
+  for (( index=0; index < ${#rvm_outputs[@]}; index++ )); do
+    __rvm_output=${rvm_outputs[$index]}
+
+    _lp_ruby_env
+    assertEquals "rvm returns at index ${index}" "${rvm_return_strings[$index]}" "$lp_ruby_env"
```

Comment:
Might as well be clear
```suggestion
    assertEquals "rvm-prompt returns at index ${index}" "${rvm_return_strings[$index]}" "$lp_ruby_env"
```

Reply:
Thanks @Rycieos